src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -197,9 +286,23 @@ public int OnNewView(IVsTextView view)
             public int RemoveAdornments()
             {
                 _sink.Unadvise();
-                _optionService.OptionChanged -= OnOptionChanged;
 
-                AddOrRemoveDropdown(enabled: false);
+                if (_optionService != null)
+                {
+                    _optionService.OptionChanged -= OnOptionChanged;
+                    _optionService = null;
+                }
```

Comment:
Consider just calling `UpdateOptionChangedSource(workspace: null)`, because that'll run the same cleanup code.